README.md

```diff
@@ -27,21 +27,31 @@
-
-The Open Grants Program focuses on funding software development and research efforts related to **Polkadot, Kusama and Substrate**. It tries to achieve this while being:
+The Open Grants Program tries to achieve the above while being:
 
 - fully transparent,
 - simple, and
-- effective
+- effective.
+
+### Guidelines
 
-The Web3 Foundation Open Grants Program focuses on technology grants up to **USD 30k for initial grants** and **USD 100k for follow-up grants**, which are tracked **transparently** on GitHub and disbursed in **bitcoin or DAI**. For larger grants, payments in other currencies and partly private submissions, please visit our [Web3 Foundation General Grants page](https://github.com/w3f/General-Grants-Program) on GitHub.
+Anyone is welcome to apply for a grant. Projects funded through our programs are broad in scope, but our focus lies on strong technical projects that add value to the ecosystem.
 
-You can find a list of areas in the Web 3.0 Technology Stack that demand particular development [here](https://github.com/w3f/General-Grants-Program/blob/master/grants/polkadot_stack.md), as well as a list of previously accepted projects [here](https://github.com/w3f/Web3-collaboration/blob/master/grants/accepted_grant_applications.md).
+Generally, your project will have better chances to get accepted if:
+- It presents a **well-researched** or tested concept, for which ideally you are able to show some prior work.
+- You can demonstrate that the project will be **maintained** after completion of the grant, be it through an obvious commitment to the technology from your side, additional funding sources or an existing business model.
+- Your team has **proven experience** with the relevant languages and technologies and/or a strong technical background. We ask you to provide the GitHub profiles of all team members as part of your application and will examine these for past activity and code quality. Naturally, you can also link to projects on other platforms.
+- Your application is **rich in technical details** and well-defined.
+- You can clearly present how your project stands out among competitors or implements technology that doesn't exist in the ecosystem yet.
 
-Additionally the builders program [feedback board](https://feedback.parity.io/) may contain some useful ideas for grant projects.
+Additionally, it must fulfill the following requirements:
+- All code produced as part of a grant must be **open-sourced**, and it must also not rely on closed-source software for full funcionality. We prefer Apache 2.0 but the GNU GPL v3 license or Unlicense are also acceptable. 
+- We do not award grants to teams that have successfully conducted a token sale or otherwise launched their own token unless there is a clear reason for it. If this applies to you, make sure to clearly present your case and explain why you cannot self-fund this project in your application.
+- Lastly, we do not provide funding for gambling-related projects.
```

Comment:
Could we make this more general: We probably should also not fund stuff related to drugs, guns, etc.

Reply:
see https://github.com/w3f/General-Grants-Program/pull/406